robotiq_2f_gripper_control/package.xml

```diff
@@ -2,7 +2,7 @@
 <package format="2">
   <name>robotiq_2f_gripper_control</name>
   <version>1.0.0</version>
-  <description>Package to control a 2-Finger Gripper from Robotiq inc.</description>
+  <description><p>Package to control a 2-Finger Gripper from Robotiq inc.</p></description>
```

Comment:
Is this a new thing for `catkin_lint` to complain about?

Reply:
(copy-paste) yes and no, catkin_lint gives a warning if the description of the package starts with "package for" or "package to" or "this package..." saying that this is the boilerplate informaion. I am not familiar with these packages and I don't feel comfortable changing their description, then I decided to add there a new format to makes catkin_lint happy..